### PR TITLE
Backend Junior 1

### PR DESCRIPTION
**Pull request by RobertMrowiec**
- [ ] Added .md file with Junior 1 Backend Path
- [ ] Updated GIT part

PR Dependency:
https://github.com/Selleo/DevPath/pull/15

### CODE REVIEW

File: backend_developer/01_junior_I.md [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+- 📗 https://www.postgresql.org/docs/11/static/datatype-boolean.html
+- 📗 https://www.postgresql.org/docs/11/static/datatype-datetime.html
+
+#### 🎤 Interview
+
+- Tell me about the basic data types in PostgreSQL.
+
+## 📦 Linux
+
+- [UNIX file types](/service_delivery_paths/backend_developer/devops/linux.md#unix-file-types)
```
Comment (PoslinskiNet) [inline backend_developer/01_junior_I.md:433]: Broken relative links...

File: backend_developer/01_junior_I.md [hunk 2]
```diff
@@ -0,0 +1,435 @@
+# Junior - level I
+
+| English Level | Experience | Verification (pick one) | Other |
+| ---- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+|  | N/A | [check outline](https://docs.google.com/spreadsheets/d/1KhP0BHtOKEo7VnnHt81gog88lyrTV3KgPTFxY4azTts/edit#gid=493855829) |  |
+| | | |  |
+
+---
+
+## 📦 GIT CLI
```
Comment (PoslinskiNet) [inline backend_developer/01_junior_I.md:10]: Reuse it maybe what we have for git already (and or merge)? Discuss with Stev0.

File: backend_developer/01_junior_I.md [hunk 3]
```diff
@@ -0,0 +1,435 @@
+# Junior - level I
+
+| English Level | Experience | Verification (pick one) | Other |
```
Comment (PoslinskiNet) [inline backend_developer/01_junior_I.md:3]: Internal only ✂️ change with force...

**Comment by Rxbsxn** [comment]
@RobertMrowiec fix pr description

**Comment by RobertMrowiec** [comment]
Fixed @PoslinskiNet

**Comment by RobertMrowiec** [comment]
Fixed! 😀 @PoslinskiNet

**Pull request merged**